src/components/floatingButton/floatingButton.api.json

```
@@ -16,6 +16,8 @@
       "type": "number",
       "description": "The bottom margin of the button, or secondary button if passed"
     },
+    {"name": "fullWidth", "type": "boolean", "description": "Whether the buttons get the container's full with", "note": "Relevant to a vertical layout with secondary button"},
```

```suggestion
    {"name": "fullWidth", "type": "boolean", "description": "Whether the buttons get the container's full width", "note": "Relevant to a vertical layout with secondary button"},
```
In the ticket the button in the image seems to be primary, is it relevant only for secondary button?

---

Right, it's only relevant to vertical layout. I'll fix it

---

fixed